added a client

lefred opened the pull request.

Commit:
Add a client node and split client and server packages for PXC

Signed-off-by: Frederic -lefred- Descamps <lefred@percona.com>

Commit:
Merge branch 'tutorial'

client added

jayjanssen commented:
I do like the idea of this.  I'm wondering how we can make the client node optional somehow so folks with lower-powered laptops can decide not to run it.  Maybe it's a simple as just explicitly doing a 'vagrant up' on node1-3 and having the 'client' node there as optional in the modules.  Any thoughts?

Commit:
Merge branch 'master' of github.com:jayjanssen/percona-xtradb-cluster-tutorial

Conflicts:
	Readme

Signed-off-by: Frederic -lefred- Descamps <lefred@percona.com>

Commit:
add myq_gadgets rpm
fix ha proxy source file for configuration, don't use absolute patch (using /vagrant) but relative to puppet

Signed-off-by: Frederic -lefred- Descamps <lefred@percona.com>
